istio-telemetry/kiali/templates/clusterrolebinding.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: istio-kiali-admin-role-binding-{{ .Release.Namespace }}
+  name: kiali
```

Comment:
This would prevent installing kiali in multiple namespaces. But it's probably ok, we want to move all cluster-wide configs to separate and kiali is not critical.

Reply:
I believe the PR was a backwards incompatibility thing